frontend/apps/filemanager/filemanager.lua

```diff
@@ -850,66 +831,143 @@ function FileManager:cutFile(file)
     self.clipboard = file
 end
 
-function FileManager:pasteHere(file)
+function FileManager:pasteFileFromClipboard(file)
     local orig_file = BaseUtil.realpath(self.clipboard)
```

Comment:
Just asking: can we get here with _not a filepath_ in the clipboard (eg. selected text from a book)? And what would happen?

Reply:
Filemanager `self.clipboard` is not a system clipboard, currently there is no way to fill it with anything other than a filepath.